src/pages/payment-methods/_mobile-expanded-list.tsx

```diff
@@ -300,7 +311,11 @@ const MobileExpandedList = ({
                     <StyledRow jc="space-between" ai="center">
                         <StyledItemDiv>
                             <Header as="p" type="subtitle-2">
-                                {is_dp2p ? localize('More info') : localize('Reference')}
+                                {is_dp2p ? (
```

Comment:
you can create a constant for this,

const sth = pd.is_dp2p ? '_t_More info_t_' : '_t_Reference_t_' 

<Localize translate_text={sth} />